f2/src/forms/InformationForm.js

```diff
@@ -10,11 +10,22 @@ import { Stack, FormControl, VisuallyHidden } from '@chakra-ui/core'
 import { FormHelperText } from '../components/FormHelperText'
 import { FormLabel } from '../components/FormLabel'
 import { P } from '../components/paragraph'
+import { useStateValue } from '../utils/state'
+
+export const InformationForm = props => {
+
+  const [data] = useStateValue ()
+  const information = {
+    information:'',
```

Comment:
I think you want three defaults here: typeOfInfoReq, typeOfInfoObtained and tellUsMore